### ISSUE TITLE
Movement page and some random thoughts

### ISSUE THREAD

**Issue by giulnz** [LN-Zap/zap-iOS#165]
First of all i'm really happy with zap developer. 
I find out slowdown at app opening since v41 i mean not really a problem just few seconds (maybe because i have a mobile wallet installed that i don't see?)

About movement page, i find it confusing.
I know there are issue open for better Visual design that i think it's most of the problem, an easy  way to visual recognize the different payment
But also the filters system is not clear to me.
For example there is on chain/lighting but if you deactivate lightning i expect to not see payment received but with created invoice On it shows it. 
I will think something about it in the next days i know is not an easy task and it's not usuful just say that for me is confusing.  

Anyway what really is important to me is security and reliability. Thats what i prefer as priority.

**Comment by ottosuess** [comment]
i agree. the History might be the thing that needs the most UX attention right now.

If you have any Ideas how to improve it let us know 🙌
